common/religions/pandaren_religion_group.txt

```diff
@@ -103,6 +103,7 @@ pandaren_religion_group = {
 			global_levy_size = -0.1
 			
 			# Opinions of other religions
+			neptulon_worship_opinion = 25
```

Comment:
The opinion not declared.
`neptulon_worship` will have more opinion of `waterspeaker` religion than their own?

Reply:
scrapped it entirely